### ISSUE TITLE
Suggestion: create GUIs in-game

### ISSUE THREAD

**Issue by Eufranio** [ustc-zzzz/VirtualChest#7]
Simple as it sounds. You could add a **/vc|virtualchest create <number of chest rows> <name>**. That would open a GUI for you, where you can put items where you want. On close, the plugin would save that GUI to disk, and load the next time someone opens the GUI.

Also, if a player is in creative mode, would be nice to be able to edit the GUI. Move the items, remove them, add new, etc.

**Comment by yamy19** [comment]
Yes please can you add this feature would make making menues so much easier
